HumanDynamicsEstimationLibrary/include/Utils.hpp

```diff
@@ -115,15 +115,13 @@ class iDynTreeHelper::State::integrator
                   iDynTree::Vector3& dot_W_p_B,
                   iDynTree::Rotation& W_R_B,
                   iDynTree::Vector3& omega_B);
-    void getJointState(iDynTree::VectorDynSize& s,
-                       iDynTree::VectorDynSize& dot_s);
+    void getJointState(iDynTree::VectorDynSize& s, iDynTree::VectorDynSize& dot_s);
     void getBaseState(iDynTree::Vector3& W_p_B,
                       iDynTree::Vector3& dot_W_p_B,
                       iDynTree::Rotation& W_R_B,
                       iDynTree::Vector3& omega_B);
     void getJointConfiguration(iDynTree::VectorDynSize& s);
-    void getBasePose(iDynTree::Vector3& W_p_B,
-                     iDynTree::Rotation& W_R_B);
+    void getBasePose(iDynTree::Vector3& W_p_B, iDynTree::Rotation& W_R_B);
     void getBasePose(iDynTree::Transform& W_T_B);
 
     void integrate(iDynTree::VectorDynSize dot_s, double dt);
```

Comment:
This is actually quite important since it's called in the loop

```suggestion
    void integrate(const iDynTree::VectorDynSize& dot_s, const double dt);
```